packages/react/src/components/table/Table.stories.tsx

```diff
@@ -0,0 +1,934 @@
+import React, { useRef, useState } from 'react';
+import { parse, isBefore, isSameDay } from 'date-fns';
+
+import { Table } from './Table';
+import { DataTable } from './DataTable';
+import workTrial from './story-example-work-trial.json';
+import { Button } from '../button';
+import { IconTrash } from '../../icons';
+
+export default {
+  component: Table,
+  title: 'Components/Table',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars
+export const Default = (args) => {
+  const cellConfig = {
+    cols: [
+      { key: 'id', headerName: 'Not rendered' },
+      { key: 'firstName', headerName: 'First name' },
+      { key: 'surname', headerName: 'Surname' },
+      {
+        key: 'age',
+        headerName: 'Age',
+        transform: ({ age }) => {
+          return <div style={{ textAlign: 'right' }}>{age}</div>;
+        },
+      },
+      { key: 'profession', headerName: 'Profession' },
+    ],
+    indexKey: 'id',
+    renderIndexCol: false,
+  };
+
+  const rows: Array<object> = [
+    { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+    { id: 1001, firstName: 'Maria', surname: 'Sarasoja', age: 62, profession: 'Designer' },
+    { id: 1002, firstName: 'Anneli', surname: 'Routa', age: 50, profession: 'Meteorologist' },
+    { id: 1003, firstName: 'Osku', surname: 'Rausku', age: 18, profession: 'Mail Carrier' },
+  ];
+
+  const caption = (
+    <span>
+      <b>Table 1</b>: Table description
+    </span>
+  );
+
+  return (
+    <div style={{ maxWidth: '640px' }}>
+      <DataTable caption={caption} cellConfig={cellConfig} rows={rows} />
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars
+export const Dark = (args) => {
+  const cellConfig = {
```

Comment:
Have to repeat these DataTable prop initialisations for them to show in the story tab of storybook.

Reply:
Is there some way to read into variable?

Reply:
Or would a function help here? I guess not

Reply:
I think we could read a constant into a variable, but then the story tab would only show the read operation like:
`const cellConfig = commonCellConfig`